src/widgets/AutocompleteSelector.js

```diff
@@ -6,9 +6,9 @@
 // eslint-disable-next-line max-classes-per-file
 import React from 'react';
 import PropTypes from 'prop-types';
-import { SearchBar } from 'react-native-ui-components';
 import { FlatList, StyleSheet, Text, TouchableOpacity, View } from 'react-native';
 import { complement } from 'set-manipulator';
+import { SearchBar } from '.';
```

Comment:
A simple suggestion here, replacing this import line for: `import { SearchBar } from './SearchBar';`

Reply:
Thanks, done!